### ISSUE TITLE
The server crashes after ~80 consecutive measures, reporting a UHD error (failed to sync with DAC probably)

### ISSUE THREAD

**Issue by LorenzoMinutolo** [LorenzoMinutolo/GPU_SDR#3]
Using UHD 3.13 boost X.68. Relaunching the server solves the problem, there is no need of restarting the USRP. Meaning that a reinitialization of the USRP would probably solve that (with the consequent loss of the phase lock between DAC and mixer)

**Comment by mkid-caltech** [comment]
![sync_t](https://user-images.githubusercontent.com/21132161/65087401-5bb72200-d96a-11e9-9344-4a3404562939.png)
Server initialization recognizes sync_t = True on UBX

**Comment by LorenzoMinutolo** [comment]
Thanks, that's a progress. Next step is to check that the timed commands for tuning TX and RX at the same time result to a reproducible phase between the two RF chains. I'll post new material here.

**Comment by LorenzoMinutolo** [comment]
the 1da418c commit should solve this problem. I tested a sequence of 100 measures without crashes. The bug was caused by the stream->reset() method being called after every measure. Removing this line could result in packet from the previous measure getting stuck in the RX buffer on the FPGA and thus varying the delay between the first and the subsequent measures. I'll test this before closing the issue.
